src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Filters } from '../../types/Filters';
+import { deleteTodo } from '../../api/todos';
+
+type Props = {
+  todos: Todo[],
+  setTodos: (todos: Todo[]) => void,
+  filter: Filters,
+  setFilterBy: (item: Filters) => void,
+};
+
+export const Footer: FC<Props> = ({
+  todos,
+  setTodos,
+  filter,
+  setFilterBy,
+}) => {
+  const activeTodosLenght = todos.filter(todo => !todo.completed).length;
+  const completedTodos = todos.filter(todo => todo.completed);
+  const hasSomeTodos = todos.some(todo => todo.completed);
```

Comment:
const hasSomeTodos = completedTodos.length > 0. Might be better for reading the code